docs/reference/query-languages/esql/esql-lookup-join.md

```diff
@@ -26,25 +28,31 @@ For example, you can use `LOOKUP JOIN` to:
 * You want to restrict users to use only specific lookup indices
 * You do not need to match using ranges or spatial relations
 
-## How the `LOOKUP JOIN` command works [esql-how-lookup-join-works]
+## How the command works [esql-how-lookup-join-works]
+
+The `LOOKUP JOIN` command adds fields from the lookup index as new columns to your results table based on matching values in the join field.
+
+The command requires two parameters:
+- The name of the lookup index (which must have the `lookup` [`index.mode setting`](/reference/elasticsearch/index-settings/index-modules.md#index-mode-setting))
+- The name of the field to join on
 
-The `LOOKUP JOIN` command adds new columns to a table, with data from {{es}} indices.
+```esql
+LOOKUP JOIN <lookup_index> ON <field_name>
+```
 
 :::{image} ../images/esql-lookup-join.png
-:alt: esql lookup join
+:alt: Illustration of the `LOOKUP JOIN` command, where the input table is joined with a lookup index to create an enriched output table.
 :::
 
-`<lookup_index>`
-: The name of the lookup index. This must be a specific index name - wildcards, aliases, and remote cluster references are not supported. Indices used for lookups must be configured with the [`lookup` index mode](/reference/elasticsearch/index-settings/index-modules.md#index-mode-setting).
+## Example
 
-`<field_name>`
-: The field to join on. This field must exist in both your current query results and in the lookup index. If the field contains multi-valued entries, those entries will not match anything (the added fields will contain `null` for those rows).
+If you're familiar with SQL, `LOOKUP JOIN` has left-join behavior. This means that if no rows match in the lookup index, the incoming row is retained and `null`s are added. If many rows in the lookup index match, `LOOKUP JOIN` adds one row per match.
 
-## Example
+### Sample tables
 
-`LOOKUP JOIN` has left-join behavior. If no rows match in the lookup index, `LOOKUP JOIN` retains the incoming row and adds `null`s. If many rows in the lookup index match, `LOOKUP JOIN` adds one row per match.
+In this example, we have two sample tables, `employees` and `languages_non_unique_key`. The `employees` table contains employee information, including their language code. The `languages_non_unique_key` table contains language codes and their corresponding names and countries.
```

Comment:
out of scope: `languages_non_unique_key` is a weird name for an index IMO; it makes sense in the context of the tests that this is taken from, but doesn't mean much to readers of this page, I think.

This extends to the example query below, which doesn't provide a meaningful example from real life. It'd be nicer to have an example that is intuitive for new users of this command.

The same applies to some of the examples in the reference entry (in `lookup-join.md`) that were taken directly from our test. The threat hunting examples are much better, in contrast.

Reply:
I agree, I think we should just make the example on the landing page be super simple. I can do that in this PR. Then we can think about how to improve the examples in the reference entry in a follow up and/or think about more end-to-end tutorials.

Reply:
I didn't want to blow the scope of this PR, but if you'd like to add a better example right away, that's great! I'm happy either way :)

Reply:
I think we can work with what we have just make it a little clearer and user-friendly :)

Reply:
for folks following along: https://github.com/elastic/elasticsearch/pull/127215/commits/9d23f244c7e200ff279ff5f2d245c9d5a53db50c